assets/js/components/coverage/CoverageMainTab.jsx

```diff
@@ -1,10 +1,81 @@
 import React, { useEffect } from "react";
 import { Link } from 'react-router-dom';
-import { Typography, Row, Col } from "antd";
+import { Table, Typography, Row, Col } from "antd";
+import Icon from '@ant-design/icons/lib/components/Icon';
+import startCase from 'lodash/startCase'
+import { minWidth } from "../../util/constants";
 const { Text } = Typography;
 import SelectedFlag from "../../../img/coverage/selected-flag.svg";
 
+const RedStatusSvg = () => (
+  <svg height="11" width="10">
+    <circle cx="5" cy="5" r="5" fill="#E06550" />
+  </svg>
+);
+
+const GreenStatusSvg = () => (
+  <svg height="11" width="10">
+    <circle cx="5" cy="5" r="5" fill="#70DC6D" />
+  </svg>
+);
+
+const OrangeStatusSvg = () => (
+  <svg height="11" width="10">
+    <circle cx="5" cy="5" r="5" fill="#E39355" />
+  </svg>
+);
+
 export default (props) => {
+  const columns = [
+    {
+      title: "Hotspot Name",
+      dataIndex: "hotspot_name",
+      render: (data) => startCase(data),
+    },
+    {
+      title: "Location",
+      dataIndex: "location",
+      render: (data, record) => {
+        if (record.long_city && record.short_country) {
+          return record.long_city + ", " + record.short_country
```

Comment:
state?

Reply:
guess it's off in the design but a little odd to not have state

Reply:
lol i can add it